pagerduty/resource_pagerduty_event_orchestration_path_unrouted.go

```diff
@@ -317,22 +306,12 @@ func performUnroutedPathUpdate(d *schema.ResourceData, unroutedPath *pagerduty.E
 	return nil
 }
 
-func buildUnroutedPathParent(d *schema.ResourceData) *pagerduty.EventOrchestrationPath {
-	orchPath := &pagerduty.EventOrchestrationPath{}
-
-	if attr, ok := d.GetOk("parent"); ok {
-		orchPath.Parent = expandOrchestrationPathUnroutedParent(attr)
-	}
-
-	return orchPath
-}
-
 func buildUnroutedPathStructForUpdate(d *schema.ResourceData) *pagerduty.EventOrchestrationPath {
 
-	orchPath := buildUnroutedPathParent(d)
-
-	if attr, ok := d.GetOk("parent"); ok {
-		orchPath.Parent = expandOrchestrationPathUnroutedParent(attr)
+	orchPath := &pagerduty.EventOrchestrationPath{
```

Comment:
Same as in case with the router - the parent was initialized twice so I cleaned it up a bit